03.log_and_register_model.py

```
@@ -0,0 +1,130 @@
+test_set = spark.table(f"{catalog_name}.{schema_name}.test_set_an").toPandas()
+
+X_train = train_set[num_features]
+y_train = train_set[target]
+
+X_test = test_set[num_features]
+y_test = test_set[target]
+
+# COMMAND ----------
+# Define the preprocessor for categorical features
+#preprocessor = ColumnTransformer(
+    #transformers=[('cat', OneHotEncoder(handle_unknown='ignore'), cat_features)], 
```

Ruff

Trailing whitespace

Remove trailing whitespace

(W291)